### PR TITLE
Add media entity

### PR DESCRIPTION
**Pull request by oshtaier**

### CODE REVIEW

File: airgun/entities/media.py [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+    def delete(self, entity_name):
+        """Delete media"""
+        view = self.navigate_to(self, 'All')
+        view.search(entity_name)
+        view.table.row(name=entity_name)['Actions'].widget.click(
+            handle_alert=True)
+
+
+@navigator.register(MediaEntity, 'All')
+class ShowAllMedias(NavigateStep):
```
Comment (abalakh) [inline airgun/entities/media.py]: Maybe we should use 'Medium' instead of 'Medias' as a plural form?

File: airgun/views/media.py [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+        breadcrumb_loaded = self.browser.wait_for_element(
+            self.breadcrumb, exception=False)
+        return (
+            breadcrumb_loaded
+            and self.breadcrumb.locations[0] == 'Installation Media'
+            and self.breadcrumb.read() == 'Create Medium'
+        )
+
+    @View.nested
+    class medium(SatTab):
```
Comment (abalakh) [inline airgun/views/media.py]: especially as 'medium' is already used due to UI naming :)

File: airgun/views/media.py [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+
+class MediasView(BaseLoggedInView, SearchableViewMixin):
+    title = Text("//h1[text()='Installation Media']")
+    new = Text("//a[contains(@href, '/media/new')]")
+    table = SatTable(
+        './/table',
+        column_widgets={
+            'Name': Text('./a'),
+            'Actions': Text('.//a[@data-method="delete"]'),
```
Comment (abalakh) [inline airgun/views/media.py]: Why not Button? Are we gonna enforce using buttons btw or it still remains optional?

Comment (oshtaier) [reply airgun/views/media.py]: That is probably under discussion. I would suggest to use Button for all unique buttons that can be addressed without xpath and in all places where you actually need to track events and etc.
Yeah, we can discuss that later

File: airgun/entities/media.py (hunk 1 above)
Comment (oshtaier) [reply airgun/entities/media.py]: whatever

**Pull request merged**